app/api/__init__.py

```diff
@@ -97,19 +97,19 @@
 # microlocations
 api.route(MicrolocationList, 'microlocation_list', '/microlocations', '/sessions/<int:id>/microlocations')
 api.route(MicrolocationDetail, 'microlocation_detail', '/microlocations/<int:id>',
-          '/events/<int:event_id>/microlocations', '/events/<identifier>/microlocations',
-          '/sessions/<int:session_id>/microlocation')
+          '/events/<int:event_id>/microlocations', '/sessions/<int:session_id>/microlocation')
```

Comment:
Why is `/events/<int:event_id>/microlocations` here ? This is microlocation detail. This should be in list.